internal/cluster/cib/parser.go

```diff
@@ -11,13 +11,13 @@ type Parser interface {
 	Parse() (Root, error)
 }
 
-type cibAdminParser struct {
+type AdminParser struct {
```

Comment:
This changes makes the code a bit more confusing. This struct is used to store the `cibadmin` command output. That's why it was named as it was. Maybe we could have `CibAdminParser` and not lint it?

Reply:
The go way is to express something with the name of the package and the thing you are importing, you refer to that struct with cib.AdminParser, instead of cib.CibAdminParser, in this case the name of the package already express the cib domain so in order to adhere to the code standars you should omit Cib in the struct name.

Anyway, if there are some cases in which repeating something brings a lot more clarity, we can opt for that, but I don't think this is the case. Anyway I'm not an expert of this domain, so if you think that we should keep the prefix, we can leave it as is

Reply:
In that case I would name it simply `Parser`. The `Admin` prefix doesn't make much sense if it is not coming with `Cib`.

Reply:
done